docs/arcaflow/workflows/expressions.md

```diff
@@ -15,16 +15,18 @@ This page explains the language elements of expressions.
 
 Literals represent values described in an expression, as opposed to values referenced from other sources.
 
-### String
+### String values
 
-string literals start and end with a matched pair of either single quotes `'` or double quotes `"`, and have zero or more characters between the quotes.
+Normal string literals start and end with a matched pair of either single quotes `'` or double quotes `"` and have zero or more characters between the quotes.
 
 Strings may have escaped values. The most important ones are for backslashes (`\\` for `\`) or for embedded newlines `\n`.
```

Comment:
Only "normal strings" may have escaped values, right?  (Since you introduced the word "normal" at line 20, you should probably use it again at line 22.)